.travis.yml

```diff
@@ -1,4 +1,3 @@
-dist: trusty
```

Comment:
So by default by removing all jdk/dist requirements, it takes a "latest" version of something ?

Reply:
Yes, the default dist used on travis is `xenial`, which includes Java 11!

Reply:
Ok cool !